src/format/mod.rs

```diff
@@ -244,8 +250,18 @@ pub enum Fixed {
     /// Parsing allows an optional colon.
     TimezoneOffsetZ,
     /// RFC 2822 date and time syntax. Commonly used for email and MIME date and time.
+    ///
+    /// When formatting the output will be localized according to `locale`.
+    ///
+    /// This does not output a strictly valid RFC 2822 string, even when not localized. RFC 2822 is
+    /// only defined on years 0 through 9999. We format a date outside these ranges anyway to
+    /// prevent a panic when formatting with `"%+"`.
     RFC2822,
     /// RFC 3339 & ISO 8601 date and time syntax.
+    ///
+    /// Note that if the year of the `DateTime` is outside of the range 0 through 9999 the date
+    /// while be formatted as an expanded representation according to ISO 8601. These dates are not
```

Comment:
grammar? I think you meant:

    /// Note that if the year of the `DateTime` is outside of the range 0 through 9999 then 
    /// the date will be formatted as an expanded representation according to ISO 8601.